core/tests/protractor_utils/ExplorationEditorHistoryTab.js

```diff
@@ -195,7 +195,7 @@ var ExplorationEditorHistoryTab = function() {
       /**
        * This method compares text contents of 2 version's state contents to
        * provided text contents
-       * v1 is most recent state and v2 is older state
+       * v1 is older state and v2 most recent state
```

Comment:
Can you keep the second `is` so that we have a verb?
```suggestion
       * v1 is older state and v2 is most recent state
```

Reply:
Done.